instagram/app/assets/stylesheets/photos.scss

```diff
@@ -0,0 +1,138 @@
+// Place all the styles related to the photos controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+h2 {
+	font-size: 16px;
+}
+
+#post {
+	border: 1px solid #dbc7c7;
+	margin-bottom: 10px;
+	padding-bottom: 5px;
+	background: white;
+}
+
+#postHeader{
+	background-color: white;
+	padding: 0px 10px;
+}
+
+
+#userName {
+	display: inline-block;
+	width: 50%;
+	color: #000080;
+	font-size: 80%;
+}
+
+#createdAt {
+	float:right;
+	display: inline-block;
+	color: #000080;
```

Comment:
Color literals like `#000080` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented with spaces, not tabs